js/main.js

```diff
@@ -1,382 +0,0 @@
-  createInputSection();
-  
-  // Returned Data Section
-  // Some elements below this row are targeted by the updateDisplay() function for dynamic populating
-  // Note IDs in side comments -> correspond to stateObj fields
-  createElement("appContainer", "div", ["container", "text-center"], "returned-data-section");
-  createElement("returned-data-section", "div", ["row", "bg-warning", "align-items-center"], "city-label-row");
-  createElement("city-label-row", "div", ["col", "display-5"], "city-label-display", "City");
-  createElement("returned-data-section", "div", ["row", "align-items-center"], "city-display-row");
-  createElement("city-display-row", "div", ["col", "align-items-center", "display-3"], "city-display", "CITY GOES HERE");   // city-display
-  createElement("returned-data-section", "div", ["row"], "zip-code-display-row");
-  createElement("zip-code-display-row", "div", ["col", "display-5"], "zip-code-display");                                   // zip-code-display
-  createElement("returned-data-section", "div", ["row"], "time-and-date-row");
-  createElement("time-and-date-row", "div", ["col-6", "display-5"], "showDate");
-  createElement("time-and-date-row", "div", ["col-6", "display-5"], "showTime");
-  createElement("appContainer", "div", ["row", "text-center", "bg-warning"], "temperature-label-row");
-  createElement("appContainer", "div", ["row", "text-center", "bg-warning"], "temperature-display-label-row");
-  createElement("temperature-display-label-row", "div", ["col", "display-5"], "temperature-display-label", "Temperature");
-  createElement("appContainer", "div", ["row"], "temperature-display-row");
-  createElement("temperature-display-row", "div", ["col-4", "text-center", "display-6"], "temp-Kelvin");                    // temp-Kelvin
-  createElement("temperature-display-row", "div", ["col-4", "text-center", "display-3", "bg-info"], "temp-Fahrenheit");     // temp-Fahrenheit
-  createElement("temperature-display-row", "div", ["col-4", "text-center", "display-6"], "temp-Celsius");                   // temp-Celsius
-  createElement("appContainer", "div", ["row", "text-center", "bg-warning"], "condition-label-row");
-  createElement("condition-label-row", "div", ["col", "display-5"], "current-conditions-label", "Current Conditions");
-  createElement("appContainer", "div", ["row"], "condition-display-row");
-  createElement("condition-display-row", "div", ["col", "text-center", "display-5"], "condition-display",);                 // condition-display
-  createElement("appContainer", "div", ["row"], "other-info-label-row");
-  createElement("other-info-label-row", "div", ["col", "display-5"], "other-info-label");
-  createElement("appContainer", "div", ["row"], "weather-img-row");
-  createElement("weather-img-row", "div", ["col", "text-center", "bg-info"], "weather-img-col");
-  createElement("weather-img-col", "img", [], "weather-img");                                                               // weather-img
```

Comment:
This will get better in the future when we don't have to create everything by hand.